opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -240,6 +261,7 @@ class Well {
         // BHP and THP limit
         double  bhp_hist_limit = 0.0;
         double  thp_hist_limit = 0.0;
+        bool    bhp_hist_limit_defaulted = true; // Tracks whether value was defaulted or not
```

Comment:
I see this value gets updated with WELTARG implementation. 

Should it also cover the situation that the BHP value is not defaulted through WCON keywords?

For example, there are values input for WCONPROD and WCONINJE, and also, for the historical wells, the control modes were BHP control, and there are values input for the BHPH iterm. 

For the situations above, the users specifies the BHP limits.

Reply:
In those situations we use the explicit values and not the defaults.